src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
+
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name, hidden = false) {
+    super(health, name);
+    this.hidden = hidden;
+    Animal.alive.push(this);
+  }
+
+  hide() {
+    if (this.hidden === true) {
+      this.hidden = false;
+    } else if (this.hidden === false) {
+      this.hidden = true;
+    }
```

Comment:
```suggestion
    this.hidden = !this.hidden
```
Можна зробити набагато коротше ;)